CMakeLists.txt

```diff
@@ -497,6 +499,29 @@ include_directories(api/include)
 
 add_subdirectory(api)
 
+if(WITH_OPENTRACING)
```

Comment:
Perhaps add a CI job for this.

Reply:
Sure, I think it would fit well in the cmake.maintainer.test, would that be acceptable?

Reply:
Done